profiler/tests/heavy.py

```diff
@@ -0,0 +1,39 @@
+import sys
+from io import StringIO
+
+import cellxgene_census
+import tiledb
+
+""" This is heavy read and process SOMA script sample developed for early testing of the profiler."""
+
+tiledb.stats_enable()
+tiledb.stats_reset()
+
+census = cellxgene_census.open_soma(census_version="2023-05-15")
```

Comment:
Again, wouldn't depend upon the census for tests specific to this repo. We do need a large soma experiment to run tests over, but I would make that entirely independent of the census. Even if it meant copying a census experiment to a new s3 location "owned" by this repo/team.  But it would be better still to be able to generate a "representative" experiment programmatically with dummy data.  In particular, we could have various SOMA experiments that use different schema settings, which might provide a more controlled way test performance.

Bottom line is that I think we need to think about the performance tests as a separate profiling-related task, and independent of developing the profiler itself. We could keep the `objects.py` test here for now.

Reply:
Looks like the wrong test file was deleted, right?